### PR TITLE
Add test mail functionality in admin settings

### LINKED ISSUE #3108: Admin Area: Add Test Email Settings
We need to be able to test the email settings. Please add an option to send a test email into the admin section in https://eventyay.com/admin/settings.

Create a test email in the system that will get send out.

Screenshot from 2019-06-11 09-14-33

### PR DESCRIPTION
**Pull request by CosmicCoder96**
Fixes #3108 

#### Short description of what this resolves:

image

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@niranjan94 Nested forms are a no-go, any ideas on how to approach the validation here, such that it does not interfere with the original form?

**Comment by CosmicCoder96** [comment]
@niranjan94 Ready for review now, validations are no longer intermixed for them.

File: app/templates/components/forms/admin/settings/system/mail-settings/test-email-form.hbs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div class="ui form">
+  {{#ui-accordion class='field'}}
+    <div class="title">
+      <i class="icon dropdown"></i>
+      {{t 'Send Test Email'}}
+    </div>
+    <div class="content four wide field">
+      <div class="ui action input">
+        {{input type="text" placeholder=(t 'Recipient E-mail') name='test_email' value=recipientEmail }}
+        <div class="ui secondary button" role="button" {{action 'sendTestMail'}}>
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/system/mail-settings/test-email-form.hbs]: Intentionally haven't made this a submit button, submit button triggers save of both the forms, tried using the form attribute of submit buttons, even then, both forms submitted.

**Comment by mariobehling** [comment]
Travis was still failing.

**Comment by CosmicCoder96** [comment]
@niranjan94 None of the tests failed, rather the build of the app itself.
`Cannot find module 'bluebird'` I don't think it is related to this PR. Have restarted the build.

**Comment by CosmicCoder96** [comment]
It passed.

**Pull request merged**